### PR TITLE
Add js

### PR DESCRIPTION
**Pull request by teifler**

### CODE REVIEW

File: bookmarks.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h2 class="app-card__title">Question</h2>
           /></a>
         </li>
         <li class="app-nav__list--item display-contents">
-          <a class="app-nav__link" href="./bookmarks.html"
+          <a class="app-nav__link js-activeSite" data-js="bookmarks" href="./bookmarks.html"
```
Comment (ttaannjjaa) [inline bookmarks.html:76]: Hi Tim, i think you pasted randomly here also the bookmark js attribute or maybe i dont understand this part of code atm

File: bookmarks.html [hunk 2]
```diff
@@ -87,5 +87,7 @@ <h2 class="app-card__title">Question</h2>
         </li>
       </ul>
     </nav>
+    <script src="activeSite.js"></script>
+    <script language="javascript" type="text/javascript" src="activeSite.js"></script>
```
Comment (ttaannjjaa) [inline bookmarks.html:91]: as i looked for this in an internet search machine i read that `language="javascript"` is not used any more - i think i will put this topic (including js correctly in a project) to my question list for our head coach

File: create.html [hunk 3]
```diff
@@ -62,5 +64,6 @@ <h1 class="app-grid__title">Create Question</h1>
         </li>
       </ul>
     </nav>
+    <script src="activeSite.js"></script>
```
Comment (ttaannjjaa) [inline create.html:67]: 😊 yes, here you also wrote a much shorter version to include js

File: profile.html [hunk 4]
```diff
@@ -54,11 +54,12 @@ <h3>Skills</h3>
...
         </li>
         <li class="app-nav__list--item display-contents">
-          <a class="app-nav__link" href="./profile.html"
+          <a class="app-nav__link js-activeSite" href="./profile.html"
             ><img src="../../images/profile-svgrepo-com.svg" width="40" height="40"
           /></a>
         </li>
       </ul>
     </nav>
+    <script language="javascript" type="text/javascript" src="activeSite.js"></script>
```
Comment (ttaannjjaa) [inline profile.html:63]: same as above

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
Great work - well done 👍😃

File: bookmarks.html (hunk 2 above)
Comment (teifler) [reply bookmarks.html:91]: thx for checking

**Pull request merged**